packages/devtools_app/lib/src/shared/feature_flags.dart

```diff
@@ -70,10 +73,17 @@ abstract class FeatureFlags {
 
   /// Flag to enable DevTools extensions.
   ///
-  /// TODO(https://github.com/flutter/devtools/issues/6443): remove this flag
+  /// TODO(https://github.com/flutter/devztools/issues/6443): remove this flag
   /// once extension support is added in g3.
   static bool devToolsExtensions = isExternalBuild;
 
+  /// Flag to enable the new Logging experience.
+  ///
+  /// https://github.com/flutter/devtools/issues/7703
+
+  static bool loggingImprovements =
```

Comment:
can we just set this equal to `enableExperiments`? I don't think we need a separate flag.